src/DynamoCore/Graph/Notes/NoteModel.cs

```diff
@@ -57,12 +57,13 @@ public NodeModel PinnedNode
         /// <param name="y">Y coordinate of note.</param>
         /// <param name="text">Text of note</param>
         /// <param name="guid">Unique id of note</param>
-        public NoteModel(double x, double y, string text, Guid guid)
+        public NoteModel(double x, double y, string text, Guid guid, NodeModel pinnedNode = null)
```

Comment:
This is an API breaking changes.

We will need a new constructor for this

Reply:
ok just added a new constructor